cfg/position.go

```diff
@@ -0,0 +1,92 @@
+package cfg
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/olebedev/config"
+)
+
+const (
+	positionPath = "position"
+)
+
+// Position represents the onscreen location of a widget
+type Position struct {
+	Height int
+	Left   int
+	Top    int
+	Width  int
+}
+
+// NewPositionFromYAML creates and returns a new instance of Position
+func NewPositionFromYAML(moduleName string, moduleConfig *config.Config) Position {
+	errs := make(map[string]error)
```

Comment:
Make this a struct that holds the name, the error, and the current value so the current value can be displayed in the error message.